src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -69,9 +69,11 @@ public void update(StandaloneConfig standaloneConfig, List<String> ignoreGlobLis
     }
 
     /**
-     * Checks for duplicate aliases in {@code authorDetailsToAuthorMap} and generates warnings
+     * Checks for duplicate aliases in {@code authorDetailsToAuthorMap} and generates warnings.
+     *
      * @param authorDetailsToAuthorMap
```

Comment:
Can consider adding 'Map to be checked for duplicate aliases.`

Reply:
Fixed